applications/protocols_serialization/client/src/logging_shell.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <logging/log_rpc.h>
+#include <zephyr/kernel.h>
+#include <zephyr/shell/shell.h>
+
+#include <stdlib.h>
+
+static int logging_get_crash_log_cmd(const struct shell *sh, size_t argc, char *argv[])
+{
+	int rc;
+	char buffer[256];
```

Comment:
Could it be configurable?

Reply:
Sure, I mean, this is just an application, not a generic module so I wanted to keep it simple, but I can add a Kconfig.